backend/clickhouse/migrations/000004_create_new_logs_table.up.sql

```diff
@@ -0,0 +1,28 @@
+CREATE TABLE logs
+(
+    Timestamp          DateTime64(9) CODEC (Delta, ZSTD(1)),
+    UUID               UUID CODEC (ZSTD(1)),
+    TraceId            String CODEC (ZSTD(1)),
+    SpanId             String CODEC (ZSTD(1)),
+    TraceFlags         UInt32 CODEC (ZSTD(1)),
+    SeverityText       LowCardinality(String) CODEC (ZSTD(1)),
+    SeverityNumber     Int32 CODEC (ZSTD(1)),
+    ServiceName        LowCardinality(String) CODEC (ZSTD(1)),
+    Body               String CODEC (ZSTD(1)),
+    ResourceAttributes Map(LowCardinality(String), String) CODEC (ZSTD(1)),
+    LogAttributes      Map(LowCardinality(String), String) CODEC (ZSTD(1)),
+    ProjectId          UInt32 CODEC (ZSTD(1)),
+    SecureSessionId    String CODEC (ZSTD(1)),
+    INDEX idx_trace_id          TraceId TYPE bloom_filter(0.001) GRANULARITY 1,
+    INDEX idx_secure_session_id SecureSessionId TYPE bloom_filter(0.001) GRANULARITY 1,
+    INDEX idx_res_attr_key      mapKeys(ResourceAttributes) TYPE bloom_filter(0.01) GRANULARITY 1,
+    INDEX idx_res_attr_value    mapValues(ResourceAttributes) TYPE bloom_filter(0.01) GRANULARITY 1,
+    INDEX idx_log_attr_key      mapKeys(LogAttributes) TYPE bloom_filter(0.01) GRANULARITY 1,
+    INDEX idx_log_attr_value    mapValues(LogAttributes) TYPE bloom_filter(0.01) GRANULARITY 1,
+    INDEX idx_body              Body TYPE tokenbf_v1(32768, 3, 0) GRANULARITY 1
+)
```

Comment:
while we're changing this, could we try the clickhouse default of `.025`? some [reading here](https://clickhouse.com/docs/en/guides/improving-query-performance/skipping-indexes/#bloom-filter-types) suggest better performance with a greater false positive rate if the values are less unique?

could omit the false positive rate value entirely to use the cc default

Reply:
Good idea! Would rather stick with clickhouse defaults as much as possible and we can modify later once we have a better baseline.